### ISSUE TITLE
Running examples on Windows

### ISSUE THREAD

**Issue by aeroaks** [38/plotters#5]
Great work with the library. I was trying to run the examples on my Windows machine.
_chart.exe_ is generated by running `cargo run --example chart` but I get the following error on running that exe:
`Error: BackendError(DrawingError(IoError(Os { code: 3, kind: NotFound, message: "The system cannot find the path specified." })))`

Also, running the webassembly demo locally does not work. It is not able to load `pkg/wasm-demo.js`.

**Comment by 38** [comment]
Thanks for reporting. I will try this on Windows and figure out why.


`Error: BackendError(DrawingError(IoError(Os { code: 3, kind: NotFound, message: "The system cannot find the path specified." })))`


This error should be caused by the output directory not found.  So the fix should be as easy as changing the path used by the `BitMapBackend`.

Also could you please provide more details how you run the WASM demo on your side?
Basically, you cannot open the demo directly because browser security restrictions. So you need serve the content with a HTTP server on your local.There's an script in that folder, which you should be able to run that.  But I haven't a chance to make sure it work for windows yet.

**Comment by aeroaks** [comment]
For the WASM demo, I ran the `start-server.sh` script from the examples directory. It fails on the last line where python is used. Since it was python 2 based http server. I changed it locally to `http.server` which is in python3 and ran the server. I can go to the link with that fix but do not see anything. Opening the console shows error in loading `pkg/wasm-demo.js`.

**Comment by 38** [comment]
That's odd, could you please share the modified script ?
I tried with python 3.3, it loads correctly.

Also, there should be a wasm-bindgen generated js file at `www/pkg/wasm-demo.js`.

**Comment by aeroaks** [comment]
I am using Python 3.7. 

After running the start-server.sh script with the following change in the last line
`python -m http.server`

I get the following error in the browser console in Firefox and Chrome.

```
Loading module from “http://localhost:8000/pkg/wasm_demo.js” was blocked because of a disallowed MIME type (“text/plain”).
localhost:8000
Loading failed for the module with source “http://localhost:8000/pkg/wasm_demo.js”.
```

**Comment by 38** [comment]
Hi,

I have finally got a chance to use a Window machine. And I made the change, now it works on Windows.

You can also use a batch file to start the server. `start-server.bat`

Closing the issue, feel free to reopen it if there's still some problem!

Thanks,
